src/commands/createNewProject/ProjectCreateStep/ScriptProjectCreateStep.ts

```diff
@@ -156,4 +159,18 @@ export class ScriptProjectCreateStep extends ProjectCreateStepBase {
             await fse.writeFile(funcIgnorePath, this.funcignore.sort().join(os.EOL));
         }
     }
+
+    private getHostContent(wizardContext: IProjectWizardContext, runtime: ProjectRuntime): object {
+        if (wizardContext.language === ProjectLanguage.PowerShell && runtime === ProjectRuntime.v2 && wizardContext.managedDependencies) {
+            return {
+                version: '2.0',
+                managedDependency: {
+                    enabled: true
+                }
+            };
+        } else if (runtime === ProjectRuntime.v2) {
+            return { version: '2.0' };
+        }
+        return {};
+    }
```

Comment:
This could be refactored into:

```ts
private getHostContent(wizardContext: IProjectWizardContext, runtime: ProjectRuntime): object {
    if (runtime === ProjectRuntime.v1) {
        return {};
    }

    if (wizardContext.managedDependencies) {
        return {
            version: '2.0',
            managedDependency: {
               enabled: true
            }
        };
    }

    return { version: '2.0' };
}
```

It's a bit cleaner and checks less things. Also it future proofs other languages that support managedDeps.

Reply:
Thanks for the suggestion @TylerLeonhardt. Fixed.